include/flatbuffers/flatbuffers.h

```diff
@@ -1294,9 +1300,30 @@ class Verifier FLATBUFFERS_FINAL_CLASS {
     return true;
   }
 
+  // Returns the message size in bytes
+  size_t GetSize() const {
+    uintptr_t size = upper_bound_ - buf_;
+    // Align the size to uoffset_t
+    size = (size - 1 + sizeof(uoffset_t)) & -uintptr_t(sizeof(uoffset_t));
+    if (buf_  + size > end_)
+      return 0;
+    return size;
+  }
+
+  // Gets the verifier buffer
```

Comment:
These kinds of comments are unnecessary for a method called `GetBuffer` inside a `Verifier` class.